src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -4,54 +4,84 @@ import { string } from 'prop-types';
 import styled from 'styled-components';
 import pathOr from 'ramda/src/pathOr';
 import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import {
+  GEL_SPACING,
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
 } from '@bbc/gel-foundations/spacings';
 import RadioSchedule from '@bbc/psammead-radio-schedule';
 import SectionLabel from '@bbc/psammead-section-label';
 import { Link } from '@bbc/psammead-story-promo';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
 import { ServiceContext } from '#contexts/ServiceContext';
 import processRadioSchedule from '../utilities/processRadioSchedule';
 import webLogger from '#lib/logger.web';
 
 const logger = webLogger();
 
+const RadioScheduleSection = styled.section.attrs(() => ({
+  role: 'region',
+  'aria-labelledby': 'Radio-Schedule',
+}))`
+  background-color: ${C_LUNAR};
+  padding: 0 ${GEL_MARGIN_BELOW_400PX};
+
+  /* To remove GEL Margins */
```

Comment:
Opted to add negative margins here to override margins added by [StyledFrontPageDiv](https://github.com/bbc/simorgh/blob/1adc63c91ca9b63a146a598b02e5d6c288bc52f7/src/app/pages/FrontPage/index.jsx#L35)

Reply:
Hey! How does moving the margins to the `FrontPageSection` behave, instead of using negative margins here?

Reply:
With the way I had tried to implement it, moving the margins to the FrontPageSection caused all of the sections to move to the right side of the page because I was moving/overriding `margin: 0 auto` as well.

Reply:
I see what you mean. What if we keep `margin: 0 auto` in front page section, remove the margin from the 'FrontPageDiv', and add that as padding to the front page section instead?

Reply:
Adding padding to `StyledFrontPageSection` pushes the section content inwards making it smaller. I think this might be because of the max-width set

<img width="1680" alt="Screenshot 2020-03-18 at 20 23 31" src="https://user-images.githubusercontent.com/12303856/76988881-86545880-6956-11ea-9295-089e5e0bdb9b.png">